### PR TITLE
Update Mothership Zeta Crew

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
I also added a bunch of relevant bashtags. This mod does changes to navmeshes in cells it doesn't otherwise touch but i didn't know whether or not to count them as wild edits

### CODE REVIEW

**Comment by MacSplody** [comment]
> I also added a bunch of relevant bashtags. This mod does changes to navmeshes in cells it doesn't otherwise touch but i didn't know whether or not to count them as wild edits

I think you'd have to resave the file in the creation kit if you remove navmesh manually.
That's getting a bit too complex for the average user.

**Comment by Superfly-Johnson** [comment]
Yeah, shame. I suppose the best thing would be to contact the mod author but I think he's retired now.

**Comment by MacSplody** [comment]
> Yeah, shame. I suppose the best thing would be to contact the mod author but I think he's retired now.

They have open permission for bug fixes, so making a replacer plugin should be fine.
The other problem is mods that require this file, do any of those contain the same edits.
Could be a little work or a lot.

**Comment by MacSplody** [comment]
Is this the Navmesh you're talking about?
That looks okay to me, it's adjacent to the other navmesh they change.

image

**Comment by Superfly-Johnson** [comment]
> Is this the Navmesh you're talking about? That looks okay to me, it's adjacent to the other navmesh they change.
> 
> image

Indeed it is, I didn't know whether or not to consider it a wild edit since it's a cell they don't directly change

**Comment by MacSplody** [comment]
> Indeed it is, I didn't know whether or not to consider it a wild edit since it's a cell they don't directly change

It's fine if the cells are next to eachother, it can happen if there are changes to nevmesh near the edge of the cell.
So some connection between the two cells changed in this case.

**Comment by Superfly-Johnson** [comment]
I'll just leave it out then, and if there's conflicts I'll put the load order data to set whatever conflicts after mothership zeta crew. I also dmed the mod author about possibly cleaning his mod, though I doubt I'll get a response

File: masterlist.yaml [hunk 1]
```diff
@@ -7448,14 +7448,30 @@ plugins:
...
+      - C.Encounter
+      - Names
+      - ObjectBounds
+      - Scripts
+    msg:
+      - <<: *wildEdits
+        subs: [ 'Delete **0500DBAA** & **0500DBAB** & **0500DBAC** & **0500DBAF** in **Mothership Crew.esm**.' ]
+        condition: 'checksum("Mothership Crew.esm", 93255384) or checksum("Mothership Crew.esm", A3370628)'
     dirty:
+      # v1.66
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

**Review by MacSplody — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -7448,14 +7448,29 @@ plugins:
...
-    url: [ 'https://www.nexusmods.com/fallout3/mods/8747/' ]
+    url:
+      - link: 'https://www.nexusmods.com/fallout3/mods/8747/'
+        name: 'Mothership Zeta Crew'
+    tag:
+      - Actors.ACBS
+      - Actors.Factions
+      - C.Encounter
+      - Names
+      - ObjectBounds
```
Comment (MacSplody) [inline masterlist.yaml]: Objectbounds aren't patched for cell/wrld records.
```suggestion
```

**Review by MacSplody — APPROVED** [review]

**Pull request merged**